lib/models/messages/twitch/eventsub_configuration.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:flutter/material.dart';
+
+class EventSubConfigurationModel extends ChangeNotifier {
+  bool showFollowEvent = true;
+  bool showCheerEvent = true;
+  bool showSubEvent = true;
+  bool showGiftedSubEvent = true;
+  bool showChannelPointEvent = true;
+  bool showHypeTrainEvent = true;
+  bool showPredictionEvent = true;
+  bool showPollEvent = true;
```

Comment:
Should we add raids here too? I think they follow the same concept as these events.